Lib/test/support/__init__.py

```diff
@@ -3040,6 +3040,19 @@ class catch_unraisable_exception:
     """
     Context manager catching unraisable exception using sys.unraisablehook.
 
+    The the *object* attribute of the unraisable hook is set and the object is
+    being finalized, the object is resurected because the context manager
+    stores a strong reference to it (cm.unraisable.object).
+
+    Storing the exception value (cm.unraisable.exc_value) creates a reference
+    cycle. The reference cycle is broken explicitly when the context manager
+    exists.
```

Comment:
Should "exists" be "exits" as I understand that reference cycle is broken explicitly when `__exit__` of context manager is called